internal/util/vault.go

```diff
@@ -136,6 +149,35 @@ func InitVaultKMS(kmsID string, config, secrets map[string]string) (EncryptionKM
 	return kms, nil
 }
 
+func parseVaultToken(kms *VaultKMS, config, secrets map[string]string) error {
+	var ok bool
```

Comment:
use standard coding guidelines/style for multiple variables (I wonder why linters did not complain about this):
```
var (
    ok bool
    err error
    token, authToken, vaultTokenFromSecret string
)
```